framework/src/modules/chain/synchronizer/block_synchronization_mechanism.js

```diff
@@ -113,20 +115,47 @@ class BlockSynchronizationMechanism {
 			);
 		}
 
-		try {
-			await deleteBlocksAfterHeight(
-				this.processorModule,
-				this.blocks,
-				lastCommonBlock.height,
-			);
-		} catch (e) {
-			this.logger.debug(
-				`Failure deleting blocks after height ${
-					lastCommonBlock.height
-				} while reverting chain to last common block`,
-			);
-			throw e;
+		await deleteBlocksAfterHeightAndBackup(
```

Comment:
Not 100% sure about the naming here? `AndBackup`?

Reply:
This is after @shuse2's comment, to specify that blocks are being backed up to temp table.